streaming/vision/convert/imagenet.py

```diff
@@ -144,14 +144,15 @@ def main(args: Namespace) -> None:
         check_extensions(filenames, extensions)
         classes, class_names = get_classes(filenames, class_names)
         indices = np.random.permutation(len(filenames))
-        if args.progbar:
+        if args.progress_bar:
             indices = tqdm(indices, leave=args.leave)
-        split_dir = os.path.join(args.out_root, split)
-        with MDSWriter(local=split_dir,
+        out_split_dir = os.path.join(args.out_root, split)
+        with MDSWriter(out=out_split_dir,
                        columns=columns,
                        compression=args.compression,
                        hashes=hashes,
-                       size_limit=args.size_limit) as out:
+                       size_limit=args.size_limit,
+                       progress_bar=args.progress_bar) as out:
```

Comment:
have you run the script with these changes?

looks like you are instantiating two progress bars at the same time